templates/snap-details.html

```diff
@@ -29,14 +29,28 @@
           <div class="p-media-object__details">
             <h1>{{ snap_title }}</h1>
             <p class="p-media-object__content--large">{{ publisher }}</p>
-
-            <p class="p-media-object__content--large">
-              {% if is_linux %}
-                <a class="p-button--positive" href="snap://{{ package_name }}" onclick="if(typeof ga !== 'undefined'){ga('gtm1.send', {hitType: 'event', eventCategory: 'Snap details', eventAction: 'Clicked Install button', eventLabel: 'Install {{ package_name }}'})}">Install</a>
-              {% else %}
-                You can install this software on a snap-powered system like <a class="p-link--external" href="https://www.ubuntu.com/download">Ubuntu</a>.
-              {% endif %}
-            </p>
+            <div class="row p-media-object__content--large">
```

Comment:
Not sure if it's common Vanilla pattern to add grid classes (like `row` here) to elements of other patterns.
But I guess it's the easiest way to achieve what we want without writing additional styling...

Reply:
Yeah, the media object pattern has some strange quirks. I want to investigate a better way of doing this kind of thing outside of this project to try and propose an improvement.